tests/fixtures/bad.yaml

```diff
@@ -0,0 +1,10 @@
+platform:
+  WORKFLOW_MANAGER: 20
+  NCORES_PER_NODE: forty
+  SCHED: slurm
+  CCPA_OBS_DIR: /scratch1/NCEPDEV/nems/role.epic/UFS_SRW_data/develop/obs_data/ccpa/proc
+  MRMS_OBS_DIR: /scratch1/NCEPDEV/nems/role.epic/UFS_SRW_data/develop/obs_data/mrms/proc
```

Comment:
I'd suggest using `scratchy` here and below so that we could make sure this is bad on all our platforms. This might exist on Hera, for example.

Reply:
@christinaholtNOAA What is `scratchy`?

Reply:
The name "scratchy" is a planned typo - the name of a disk which does not exist on any of the HPC platforms.

Reply:
Oh, `scratchy` instead of e.g. `scratch1`. I like it.